src/main/java/io/codurance/pouch/ResourceFavouriteController.java

```diff
@@ -0,0 +1,57 @@
+package io.codurance.pouch;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+class ResourceFavouriteController {
+
+    private final ResourceRepository ResourceRepository;
```

Comment:
Should the member variable be named with a leading lower case `r`? Reading this file just seemed odd to me and makes me think we have static methods on the member.